.codecov.yaml

```diff
@@ -11,15 +11,17 @@ coverage:
   status:
     project:
       default:
-        enabled: yes
         target: 90%
         paths:
-          - api
-          - sdk
+          - "!opencensus-shim/"
+          - "!opentracing-shim/"
+          - "!sdk-extensions/zpages/"
+      unmaintained:
+        target: 80%
+        paths:
+          - "opencensus-shim/"
```

Comment:
FYI - Myself (and my team) are maintaining this and coordinating releases with OpenCensus.   So far, we haven't done much because we've been focused on OTel MEtrics SDK stabilization before continuing here.

Reply:
#3829 Actions >> words